### DISCUSSION TITLE
Bluetooth flyout on Windows 10 taskbar

### DISCUSSION THREAD

**Discussion by codeBodger** [valinet/ExplorerPatcher#2352] [Ideas]
Under System tray in Properties, there's the option to use either the Windows 10 flyout or Windows 11 flyout (or other options) for many of the system icons, however there is not this option for Bluetooth.  The menu that appears when clicking on the Bluetooth icon in the system tray is usable, but is really only useful to open settings, as there's no way to connect to a Bluetooth device otherwise.  The Windows 11 flyout has much better functionality, but there's no option to use it with the Windows 10 taskbar.  

I would greatly appreciate it if this would be added.

**Comment by pyrates999** [comment]
Unless Microsoft supports such a flyout in the windows code, I don't see this happening.  What would it look like exactly?

**Comment by Amrsatrio** [comment]
<img width="54" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/20662640/98cc7aab-2abe-40e7-b955-de6175967918">

I mean it's theoretically possible to change the behavior of clicking this into opening that little Bluetooth devices list in the Win11 control center. But I'm just not in the mood to do new mods at the moment.

**Comment by pyrates999** [comment]
Here's how you can do it if your windows version supports it: https://www.youtube.com/watch?v=TrZfSmLw_m4

I have not tested it, so no guarantee it will still work.

**Comment by akamienski** [comment]
Is it actually in works? Would love to have this feature possible like in StartAllBack.

**Comment by pyrates999** [comment]
You can check to see if a mod in windhawk can do this.
